### ISSUE TITLE
Training for CCTV system

### ISSUE THREAD

**Issue by acalatrava** [AlexeyAB/darknet#1647]
I'm a beginner on DNN but I've read a lot and I think I'm doing everything "by the book". However I think my training is not working well. I want to create a training for my CCTV system so I think it should be pretty easy to train. I have 6 cameras and I'm using a feed with only VGA resolution (640x480).  I only want to detect 1 class to start with (person) and I walk around the cameras to get some frames and mark them (using YoloMark). 
I marked a bunch of them (like 400 or so) and follow your guide to start a training on a multigpu environment. However during the training I always get NAN on Region 82 and Region 94. And after 300 or 400 iterations I get NAN on Region 106 too. (I don't know what those Regions means, btw...)

Also I can see that every iteration the AVG LOSS is increasing instead of decreasing... I read somewhere that I may need to wait a couple of thousand iterations but since I'm only seeing NANs I'm not really sure about that...

BTW I used calc_anchors too to set it on the config file...

This is the config file

```
[net]
# Testing
batch=64
subdivisions=64
# Training
batch=64
subdivisions=8
width=608
height=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=1000
max_batches = 500200
policy=steps
steps=400000,450000
scales=.1,.1
```

Thank you for your time!

**Comment by SouradipBh** [comment]
Hi @acalatrava
Have you connect cctv camera with darknet?
If it is Yes, then which part you have make changes?
